### ISSUE TITLE
Invalid Year

### ISSUE THREAD

**Issue by ecarrera** [mathieustan/vue-datepicker#47]
Hi @mathieustan, I'm currently in Chile (GMT-3) and I think that the GMT difference is giving me a bad format conversion for the year.

I took, by example a year and the shown year is 1 less than the chosen.

This is the implementation:

```javascript
<vue-date-picker
  v-model="fieldData.value"
  v-bind="fieldData.settings"
  :name="fieldData.label"
  :type="fieldData.fieldType"
  :fullscreen-mobile="true">
  <template #activator="{date}">
    <button
      :id="key"
      :name="fieldData.label"
      :placeholder="fieldData.placeholder"
      class="form-control form-control-lg d-flex justify-content-between align-items-center">
      {{ date ? date : '&zwnj;' }}
      <font-awesome-icon
        icon="calendar"
        class="text-muted"
        size="sm" />
    </button>
  </template>
</vue-date-picker>
```

If I don't use a template the year is fine, but, once the calendar is opened again the selected year on the calendar is 1 year less than the real selection.

Can it be a bug? Am I doing something wrong? Or there is a way to avoid GMT conversions?

Greets.

**Comment by mathieustan** [comment]
Hello @ecarrera. 

I tried different things, and it seems to work for me. Can you give me a reproduction link ?

**Comment by ecarrera** [comment]
![ezgif-4-4f06c7db4f5b](https://user-images.githubusercontent.com/2139735/83911791-e3cc9a00-a73a-11ea-8e17-f7fc7629581d.gif)

https://codesandbox.io/s/peaceful-nobel-cssqq

**Comment by mathieustan** [comment]
Fixed in [v0.2.5](https://github.com/mathieustan/vue-datepicker/releases/tag/0.2.5)
